pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributorsTest.scala

```diff
@@ -461,18 +338,31 @@ class SierraContributorsTest
         expectedContributors = expectedContributors)
     }
 
-    it("gets the name from MARC tag 710 subfield $$a") {
-      val name = "Karl the kale"
+    it(
+      "combines only subfields $$a $$b $$c $$d (multiples of) with spaces from MARC field 110 / 710") {
+      // Based on https://search.wellcomelibrary.org/iii/encore/record/C__Rb1000984
+      // as retrieved from 25 April 2019
+      val name =
+        "IARC Working Group on the Evaluation of the Carcinogenic Risk of Chemicals to Man."
+      val subordinateUnit = "Meeting"
+      val date = "1972 :"
+      val place = "Lyon, France"
 
       val varFields = List(
         createVarFieldWith(
-          marcTag = "710",
-          subfields = List(MarcSubfield(tag = "a", content = name))
+          marcTag = "110",
+          subfields = List(
+            MarcSubfield(tag = "a", content = name),
+            MarcSubfield(tag = "b", content = subordinateUnit),
+            MarcSubfield(tag = "d", content = date),
+            MarcSubfield(tag = "c", content = place)
```

Comment:
Worth adding another subfield with a different tag and asserting that it gets ignored?

Reply:
Done. I'd like to run through these tests too as it feels like there might be some redundancy about as we're simplifying things, but also don't want to move to many parts.